packages/release-manifests/src/manifest.ts

```diff
@@ -15,6 +15,7 @@
  */
 
 import fetch from 'cross-fetch';
+import 'global-agent/bootstrap';
```

Comment:
Seeing as this is global I think it would be better if we add this to the CLI instead, since this is an isomorphic package.

Reply:
So I will add the import to https://github.com/backstage/backstage/blob/master/packages/cli/src/index.ts incl. the yarn dependency?

Reply:
If possible import it in https://github.com/backstage/backstage/blob/master/packages/cli/src/commands/versions/bump.ts and add the dependency to https://github.com/backstage/backstage/blob/master/packages/cli/package.json

Reply:
Change committed.